android/quest/src/main/assets/configurations/view/detail_view_configurations.json

```diff
@@ -107,6 +111,56 @@
             }
           }
         ]
+      },
+      {
+        "filters": [
```

Comment:
this and below look too specific to not be loaded via a dynamic config

Reply:
This one load PQ regimen recommended for patient which is 8.0 weeks for deficient, and intermediate g6pd and  2.0 week for normal, and N/A for others criteria. This is determined by CQL and ots values could only be 8.0/2.0/Missing.  86859003 coded medication request know what is the value for this. dosageInstruction.timing is the data we want to show on UI. This can change to anything else into resource properties path.

Below this config we are displaying info text about why a value is shown i.e. what criteria makes it 8.0/2.0/Missing which is also determined by the value of 86859003 coded med-request. the corresponding message is picked from list based on the data we find for patient.

Reply:
https://user-images.githubusercontent.com/4829880/150308380-9ee3f1c9-c2a9-4ac2-b902-db2e208f8498.jpg in this image 3.0, 9.0, Deficient, N/A is loaded from data and everything else is the label associated with data. The label can change according to values which have predictable values

Reply:
that all makes sense, my main point is that we don't want these hard-coded in the Quest app, we want to load them into it dynamically

Reply:
This can then moved to note into medication-request or text field into med-request.dosageInstructions. This intent to keep this as Label was to allow keeping it closest to presentation layer so that we can change the  text easily when ever needed.. otherwise if it is in patient data any update would require data update for all..

E.g. current text on screen is 
**'People with G6PD status Deficient are given 8 Weeks regimen'.**
Above can be changed to 
**'Patient having G6PD Deficiency are usually given 8 Weeks medicine. Make sure to follow up with your physician after 9 weeks'.**
without any changes to data rather its a change into config. otherwise older data would need update OR we would have different data for different tests...

Reply:
These are already in configs which for now reside into assets and would be moved into Composition. So yeah with ticket mentioned this would exist on server along with all other configs

Reply:
OK that is sufficient, I'm certainly not suggested we mix this with patient data.

I wanted to clarify that we could move this to the loaded composition, thank you for explaining that to me. This path makes sense, we can take out the g6pd key in this config file, move it into the app defining Composition that is loaded synced on first load and then load from there, removing the g6pd specific portion of this configuration.